cfme/containers/provider/openshift.py

```diff
@@ -137,6 +158,13 @@ def __init__(
             endpoints=endpoints,
             appliance=appliance)
 
+    def create(self, **kwargs):
+
+        if getattr(self, "alerts_type", None) == "Prometheus":
+            self.enable_all_alerts_collection()
+
+        super(OpenshiftProvider, self).create(**kwargs)
```

Comment:
Yep, this is exactly what I was thinking with moving this out of the common provider class.  Let me know how testing goes, if its working the same as it was when it was in the common class's method (it should!)

Reply:
That's works the same way...